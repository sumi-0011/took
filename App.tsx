import React from 'react';
import {NativeBaseProvider} from 'native-base';
import {NavigationContainer} from '@react-navigation/native';
import MainNavigation from '~/navigations/MainNavigation';

function App() {
  return (
    <NativeBaseProvider>
      <NavigationContainer>
        <MainNavigation />
      </NavigationContainer>
    </NativeBaseProvider>
  );
}

export default App;
