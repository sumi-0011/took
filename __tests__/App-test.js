import 'react-native';
import React from 'react';
import App from '../App';

import {render} from '@testing-library/react-native';
let props;
let component;

function getComponent(props) {
  return <App {...props} />;
}

describe('App test...', () => {
  props = {};
  component = getComponent(props);
  test('test 1', () => {
    const rendering = render(component);
    expect(rendering).toMatchSnapshot();
    expect(rendering).toBeTruthy();
  });
});
