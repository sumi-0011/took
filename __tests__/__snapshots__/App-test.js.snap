// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App test... test 1 1`] = `
<RNCSafeAreaProvider
  onInsetsChange={[Function]}
  style={
    Array [
      Object {
        "flex": 1,
      },
      undefined,
    ]
  }
/>
`;
